AppCenterData/AppCenterData/Internal/Client/MSTokenExchange.m

```diff
@@ -79,7 +79,7 @@ + (void)performDbTokenAsyncOperationWithHttpClient:(id<MSHttpClientProtocol>)htt
 
             // Token store returned non-200 response code.
             if (response.statusCode != MSHTTPCodesNo200OK) {
-              MSLogError([MSData logTag], @"The token store returned %ld", (long)response.statusCode);
+              MSLogWarning([MSData logTag], @"The token store returned %ld", (long)response.statusCode);
```

Comment:
This could hide some other issues. If 404 could be expected, can you log warning message only for 404? I guess it could be not a warning level but something lower than warning.

Reply:
@jaelim-ms I agree.  Do we have other logic that splits out error codes by number and gives different log levels based on the number, though?  I think at worst this case is an error, so if we want it to cover the worst number codes, it should remain an error.  If we want to make it less confusing, which is @MehrdadMzfr's goal, we should make more granular responses

Reply:
reverted this change.